libvirt/tests/src/virtual_network/qemu/netperf.py

```
@@ -0,0 +1,901 @@
+                recreate_serial_console=recreate_serial_console,
+                restart_network=True
+            ).close()
+        except virt_vm.VMIPAddressMissingError:
+            pass
+
+    session = vm.wait_for_login(timeout=login_timeout)
+
+    test.log.debug(
+        "TEST_STEP 5: Configure guest NIC (queues/IP) and collect addresses")
+    mac = vm.get_mac_address(0)
+    data_mac = vm.get_mac_address(1) if len(nics.split()) > 1 else mac
```

Ruff (0.14.14)

[error] Local variable `data_mac` is assigned to but never used

Remove assignment to unused variable `data_mac`

(F841)